### PR TITLE
Meta Learning blog

### PR DESCRIPTION
**Pull request by anand-kummari**
blog on meta learning

### CODE REVIEW

File: source/_posts/Meta-Learning-Learning-to-learn-fast.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
+---
+title: 'Meta-Learning:Learning-to-learn-fast'
```
Comment (Rajeev111974) [inline source/_posts/Meta-Learning-Learning-to-learn-fast.md:2]: title tag should be 50-60 character & it should be in Heading tag

File: source/_posts/Meta-Learning-Learning-to-learn-fast.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+here one dataset is considered as one sample data point.
+
+### **Few-shot classification**
+
+*Few-shot classification* is an instantiation of meta-learning in the field of supervised learning. The dataset ***D*** is often split into two parts, support set ***S*** for learning and a prediction set ***B*** for training or testing, ***D=*(*S, B*)**. Often we consider a *K-shot N-class classification* task: the support set contains K labeled examples for each of N classes.
+
+{% asset_img image.png  An example of a 4-shot 2-class image classification %}
+
+<!-- ![*An example of a 4-shot 2-class image classification.*](https://cdn-images-1.medium.com/max/2000/1*NzBnn1dqwh_4_eH9vYVj4w.png)**An example of a 4-shot 2-class image classification.** -->
```
Comment (Rajeev111974) [inline source/_posts/Meta-Learning-Learning-to-learn-fast.md:44]: Optimize your images with image alt text

File: source/_posts/Meta-Learning-Learning-to-learn-fast.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
+---
+title: 'Meta-Learning:Learning-to-learn-fast'
+date: 2020-05-06 10:36:34
+author: Anand Kummari
+category: AI
+tags:
+- Machine Learning
+- Meta Learning
```
Comment (Rajeev111974) [inline source/_posts/Meta-Learning-Learning-to-learn-fast.md:8]: tag should be linked with main website

File: source/_posts/Meta-Learning-Learning-to-learn-fast.md (hunk 1 above)
Comment (anand-kummari) [reply source/_posts/Meta-Learning-Learning-to-learn-fast.md:2]: changed title to Meta-Learning: Learning-to-learn-fast and its approaches.
If I write title in heading, I can see it twice,so keeping it in title

File: source/_posts/Meta-Learning-Learning-to-learn-fast.md (hunk 2 above)
Comment (anand-kummari) [reply source/_posts/Meta-Learning-Learning-to-learn-fast.md:44]: optimized images and added description for images. few images are formulas,so did not add Alt for them

File: source/_posts/Meta-Learning-Learning-to-learn-fast.md (hunk 3 above)
Comment (anand-kummari) [reply source/_posts/Meta-Learning-Learning-to-learn-fast.md:8]: tagged nayan and nayanindia

**Comment by Rajeev111974** [comment]
<meta name="keywords" content="" /> keyword is not there, it should be there  

<meta name="description" content="Installing Tensorflow 2.0 on Ubuntu 18.04 using docker. Run all experiments from a container.I have come across many developers who face serious issues when it comes to installing tensorflow on a linu"> : Content character should be 160 chr.

Keyword should be linked with main website so that it can redirected to the web

Feature image is not there it should be there

1–2 long-tail keywords should be there in the content

Previous blog link should be there in the blog.

Blog Post Heading H1 should be there :  right now heading is not in H1 tag
    
Add targeted kyeword in url

Title tag should be 50-60 charcter : right now 30 character

Optimize the length of your article. 

Make sure your articles have a minimum of 300 words. Google likes long articles, however, if your article is too long – and not so easy to read – it might scare users away. Only try writing long articles when you know you're a skilled writer

Write Optimized URL (example below)

http://www.example.com/category-keyword/subcategory-keyword/primary-keyword.html

Add Video Content there in the blog.

Categories should be mentioned there.

File: source/_posts/Meta-Learning-Learning-to-learn-fast.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
+---
```
Comment (Rajeev111974) [inline source/_posts/Meta-Learning-Learning-to-learn-fast.md:1]: Keyword is not linked with Website

Comment (Rajeev111974) [inline source/_posts/Meta-Learning-Learning-to-learn-fast.md:1]: in content somewhere link should be there with previous blog

Comment (Rajeev111974) [inline source/_posts/Meta-Learning-Learning-to-learn-fast.md:1]: keyword should be linked with main website

Comment (Rajeev111974) [inline source/_posts/Meta-Learning-Learning-to-learn-fast.md:1]: add video content in the blog

Comment (Rajeev111974) [inline source/_posts/Meta-Learning-Learning-to-learn-fast.md:1]: /blog/2020/05/06/Meta-Learning-Learning-to-learn-fast/ - url is not correct it should be in text format with hyphen

**Pull request merged**